weatherApp/Features/WeatherDetails/UI/DetailWeatherPresnter.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DetailWeatherPresnter.swift
+//  weatherApp
+//
+//  Created by Dario Nikolic on 11/08/2020.
+//  Copyright © 2020 Dario Nikolic. All rights reserved.
+//
+
+class DetailWeatherPresenter {
+    
+    let currentWeather: CurrentWeatherViewModel
+    var numberOfConditions: Int {
+        weatherConditionList.count
+    }
+    var numberOfConditionRows: Int {
+        numberOfConditions / 2 + numberOfConditions % 2
+    }
+    
+    private var weatherConditionList: [ConditionInformationViewModel] = []
```

Comment:
You can move private property above public properties. Private functions are bellow public functions. This is our standard in Five.